server/controller/chore.js

```diff
@@ -38,14 +38,15 @@ export const createChore = async (req, res) => {
         message: `Chore: ${name} already exists. If you want to change somthing, please use the EDIT option.`,
       });
     }
-
+    const createdAt = Date.now();  
```

Comment:
We should probably add a date field indicated by the user on the frontend. Would allow the user to create all the chores at once without future occurrences locked to the date they made the chore on.

Reply:
The created at feild is just a time stamp for the backend.
It is used to calculate if a chore is to be added to the schedule today. Nothing more